### ISSUE TITLE
tradercopy.py isn't working

### ISSUE THREAD

**Issue by Gaspard2005** [DogeIII/Binance-Leaderboard-CopyTrading#2]
Hello I have always this error when I want to run tradercopy.py : 
Traceback (most recent call last):

  File "/home/gaspard/Téléchargements/Binance-Leaderboard-CopyTrading/trader/trader2/tradercopy.py", line 214, in <module>

    for item in data['data']['otherPositionRetList']:

TypeError: 'NoneType' object is not iterable

Do you have an idea.
If you can provide a way to reach you it will be pretty great, so we can discuss about your work, I have some idea to upgrade it.
But thanks for your work then.

**Comment by DogeIII** [comment]
> Hello I have always this error when I want to run tradercopy.py : Traceback (most recent call last):
> 
> File "/home/gaspard/Téléchargements/Binance-Leaderboard-CopyTrading/trader/trader2/tradercopy.py", line 214, in
> 
> ```
> for item in data['data']['otherPositionRetList']:
> ```
> 
> TypeError: 'NoneType' object is not iterable
> 
> Do you have an idea. If you can provide a way to reach you it will be pretty great, so we can discuss about your work, I have some idea to upgrade it. But thanks for your work then.

Certainly! Here's a revised version of your message for the GitHub issue:

Could you please provide me with the UUID of the trader you are trying to follow?

**Comment by frcd10** [comment]
Binance changed the way you scrap it. It return None if you are not logged in website.

**Comment by Gaspard2005** [comment]
was this one but I m not sure its a good one it was just to test the bot : 3AFFCB67ED4F1D1D8437BA17F4E8E5ED
But yeah in fact its what I have found by myself so if I found a way I'll tell you
